sensorimotor/experiments/capacity/defs/vary_elements.yaml

```diff
@@ -0,0 +1,20 @@
+- worlds: 2
```

Comment:
I found that experiments run faster when you run the larger conditions first. The largest condition ends up taking longer than several of the smaller conditions.  If you'd still prefer smaller first I'll change it.